lib/vsc/install/commontest.py

```diff
@@ -87,15 +93,16 @@ class CommonTest(TestCase):
     #   Whitelist: if match, fail test
     PROSPECTOR_BLACKLIST = [
         # 'wrong-import-position',  # not sure about this, these usually have a good reason
+        "Redefining built-in 'basestring'",  # basestring is defined when running on top of Python 3.x
         'Locally disabling',  # shows up when you locally disable a warning, this is the point
         'Useless suppression',  # shows up when you locally disable/suppress a warning, this is the point
+        'redefined-builtin',
```

Comment:
readd it to the whitelist

Reply:
sorry to be clear - it originally was on the blacklist, and you want it removed from the blacklist and added instead to the white list?